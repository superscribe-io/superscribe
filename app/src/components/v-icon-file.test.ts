import { test, expect } from 'vitest';
import { mount } from '@vue/test-utils';

import VIconFile from './v-icon-file.vue';

test('Mount component', () => {
	expect(VIconFile).toBeTruthy();

	const wrapper = mount(VIconFile, {
		props: {
			ext: 'png',
		},
		global: {
			stubs: ['v-icon'],
		},
	});

	expect(wrapper.html()).toMatchSnapshot();
});
