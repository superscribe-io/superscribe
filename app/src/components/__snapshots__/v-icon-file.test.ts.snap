// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Mount component 1`] = `
"<div data-v-a90644d4=\\"\\" class=\\"icon\\">
  <v-icon-stub data-v-a90644d4=\\"\\" name=\\"insert_drive_file\\"></v-icon-stub><span data-v-a90644d4=\\"\\" class=\\"label\\">png</span>
</div>"
`;
